cfme/tests/services/test_rest_services.py

```diff
@@ -522,7 +523,24 @@ def test_add_service_parent(self, request, appliance):
         parent = collection.action.create(service_body())[0]
         request.addfinalizer(parent.action.delete)
         child = collection.action.create(service_body())[0]
-        child.action.edit(ancestry=str(parent.id))
+        child.action.edit(parent_service=parent._ref_repr())
+        assert_response(appliance)
+        child.reload()
+        assert child.ancestry == str(parent.id)
+
+    @pytest.mark.uncollectif(lambda: store.current_appliance.version < '5.8')
```

Comment:
Use version.current_version() here

Reply:
My understanding is that ``current_version()`` is being deprecated and will slowly go away. ``appliance.version`` is the preferred way now and since it cannot be used in decorator, ``store.current_appliance.version`` is the closest thing.

Reply:
See https://github.com/ManageIQ/integration_tests/pull/5597#discussion_r144657163

Reply:
I'm also not really sure if it is good to use store.current_appliance.version. Maybe better will be get_or_create_current_appliance().version?
@izapolsk @psav what do you think?